### ISSUE TITLE
Update Docker pr-test image

### ISSUE THREAD

**Issue by Genc** [louislam/uptime-kuma#2804]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

There has been no update for the pr-test image for 5 months. Can we update this?

![image](https://user-images.githubusercontent.com/18603640/219940377-9ae048d2-db33-4a2d-8d8d-7515f719d21a.png)

I think running these commands will enough.
* npm run build-docker-pr-test
* docker push

Might be good to automate this to run on every LTS update?

### 🐻 Uptime-Kuma Version

1.18.0

### 💻 Operating System and Arch

Docker

### 🌐 Browser

Chrome

**Comment by louislam** [comment]
Just update, but it is not important, because this image is designed for testing pr, it will checkout and install everything on the fly.

https://github.com/louislam/uptime-kuma/wiki/Test-Pull-Requests

**Comment by Genc** [comment]
> Just update, but it is not important, because this image is designed for testing pr, it will checkout and install everything on the fly.
> 
> https://github.com/louislam/uptime-kuma/wiki/Test-Pull-Requests

Yes i know, but if a person has not received a pull from the master for a long time, she will be able to test it with the old version and will not know whether the new feature breaks a different place.
